web_api/core/views.py

```diff
@@ -320,3 +320,7 @@ def sync_accounts(request: HttpRequest) -> HttpResponse:
     except SyncAccountsError:
         return JsonResponse(dict(ok=False))
     return JsonResponse(dict(ok=True))
+
+
+def debug_sentry(request: HttpRequest) -> HttpResponse:
```

Comment:
Is this still necessary?

Reply:
I think it's nice to have a quick way to verify that sentry is working.